Tests/HTTPTests/HTTPBodyTests.swift

```diff
@@ -2,12 +2,14 @@ import XCTest
 @testable import HTTP
 import Transport
 import libc
+import SocksCore
 
 class HTTPBodyTests: XCTestCase {
     static var allTests = [
         ("testBufferParse", testBufferParse),
         ("testChunkedParse", testChunkedParse),
         ("testClientStreamUsage", testClientStreamUsage),
+//        ("testReleasingServer", testReleasingServer),
```

Comment:
This test is new. It starts a server, connects to it, stops it and checks if the previously opened connection was correctly closed. For some reason, this test fails about 20% of the time in my environment. I'm not sure why and will have to look at SocksCore again. I think the PR can be merged without the new test.